### DISCUSSION TITLE
New install - asking to drop tables that don't exist?

### DISCUSSION THREAD

**Discussion by kristophr** [firefly-iii/firefly-iii#4860] [Firefly III]
I started installing firefly from scratch and was following the steps in this page:

https://docs.firefly-iii.org/firefly-iii/installation/self_hosted/

am doing the self-hosted - non-docker version with basic mysql/php/nginx set up (last year I used firefly with no issue but want to start using it again)

I got to this command:

php artisan migrate:refresh --seed

and it fails when it is creating the database.
Its trying to drop tables that don't exist? 

"table not found: firefly.attachments firefly.account_meta and others - this doesn't make sense.  Why would it drop tables that dont' exist when trying to create a new database from scratch?

**Comment by JC5** [comment]
It doesn’t. Do you have the full logs?

**Comment by kristophr** [comment]
https://pastebin.pl/view/45d134d0

Here are the last two times I ran the command

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by JC5** [comment]
Sorry, it doesn't tell me more. I don't know why it's doing this.

**Comment by JC5** [comment]
The thing is: it will only drop tables when a "migration" table is found and then, each migration in that table will be rolled back (or at least tried).

Do you have the logs from the entire install perhaps? At some point, the logs will say "migration table found / not found"

**Comment by kristophr** [comment]
Unfortuntely not – what I can do is delete everything and start again.  I can keep logs of everything that happens from that point on.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Reply by JC5** [reply]
I owuld appreciate that, thanks!
